Use gnome-open to open most file types

Instead of using a hard coded application claws will use the GNOME preferred
application.

http://issues.foresightlinux.org/browse/FL-432
diff --git a/src/common/defs.h b/src/common/defs.h
--- a/src/common/defs.h
+++ b/src/common/defs.h
@@ -118,15 +118,15 @@
 /* #define DEFAULT_INC_PATH	"/usr/bin/imget" */
 /* #define DEFAULT_INC_PROGRAM	"imget" */
 #define DEFAULT_SENDMAIL_CMD	"/usr/sbin/sendmail -t -i"
-#define DEFAULT_BROWSER_CMD	"firefox '%s'"
+#define DEFAULT_BROWSER_CMD	"gnome-open '%s'"
 #ifndef MAEMO
-#define DEFAULT_EDITOR_CMD	"gedit '%s'"
+#define DEFAULT_EDITOR_CMD	"gnome-open '%s'"
 #else
 #define DEFAULT_EDITOR_CMD	"leafpad '%s'"
 #endif
 #define DEFAULT_MIME_CMD	"metamail -d -b -x -c %s '%s'"
-#define DEFAULT_IMAGE_VIEWER_CMD "display '%s'"
-#define DEFAULT_AUDIO_PLAYER_CMD "play '%s'"
+#define DEFAULT_IMAGE_VIEWER_CMD "gnome-open '%s'"
+#define DEFAULT_AUDIO_PLAYER_CMD "gnome-open '%s'"
 
 #ifdef _PATH_MAILDIR
 #  define DEFAULT_SPOOL_PATH	_PATH_MAILDIR
